### ISSUE TITLE
regression_losses always 0

### ISSUE THREAD

**Issue by tingggggg** [zylo117/Yet-Another-EfficientDet-Pytorch#360]
as title

Step: 499. Epoch: 0/500. Iteration: 500/15703. Cls loss: 0.06598. Reg loss: 0.00000. Total loss: 0.06598:   3%|▊                        | 499/15703 [04:44<2:22:57,  1.77it/s]checkpoint...
Step: 999. Epoch: 0/500. Iteration: 1000/15703. Cls loss: 0.04717. Reg loss: 0.00000. Total loss: 0.04717:   6%|█▌                      | 999/15703 [09:23<2:16:33,  1.79it/s]checkpoint...
Step: 1499. Epoch: 0/500. Iteration: 1500/15703. Cls loss: 0.03463. Reg loss: 0.00000. Total loss: 0.03463:  10%|██                    | 1499/15703 [14:03<2:12:13,  1.79it/s]

I train on own dataset when the first step, Reg loss is 0. Cls Loss seems normal that be down step by step .
Any training parameter need to check and change to own value, or anyone can give me some advice.
Thanks, for ur help~

**Comment by 6457tang** [comment]
Have you solved the problem? I have the similar situation with you.
![image](https://user-images.githubusercontent.com/46106417/83742812-7104e580-a68c-11ea-9cbd-fac9c7c1afdd.png)

**Comment by xiangjun2** [comment]
positive_indices = torch.ge(IoU_max, 0.5)  reduce 0.5 to 0.4. Good luck

**Comment by zylo117** [comment]
> positive_indices = torch.ge(IoU_max, 0.5) reduce 0.5 to 0.4. Good luck

I don't think this is a good solution though. The root cause is the unsuitable anchors. And what you do is to reduce the impact of the unsuitable anchors, but the root cause is still there.

**Comment by zhouzhubin** [comment]
@tingggggg 你好，请问你后面是怎么解决的啊？我根据https://github.com/mnslarcher/kmeans-anchors-ratios也调整anchors，但是 Reg loss还是为 0.00000.  这个positive_indices = torch.ge(IoU_max, 0.5)， 我把0.5减到0.1才行，那个reg loss才不会为0

**Comment by lgldl** [comment]
> Have you solved the problem? I have the similar situation with you.
> ![image](https://user-images.githubusercontent.com/46106417/83742812-7104e580-a68c-11ea-9cbd-fac9c7c1afdd.png)

we also meet this problem, can you solve it?
